spec/helpers/middleman/page_spec.rb

```diff
@@ -174,7 +174,7 @@
       expect(@app.locale_page(page: 'index')[:page]).to eq('index')
       expect(@app.locale_page(page: 'blog')[:page]).to eq('blog')
       expect(@app.locale_page(page: 'pro')[:page]).to eq('pro')
-      expect(@app.locale_page(page: 'about-sharesight')[:page]).to eq('about-sharesight') # landing page
+      expect(@app.locale_page(page: 'pricing')[:page]).to eq('pricing') # landing page
```

Comment:
Weird, this isn't (and previously wasn't) a landing page…but I'll ignore it, I don't think I really care about test coverage in here as long as it's not breaking.

Reply:
sweet - removed them !